README.rst

```diff
@@ -112,6 +112,8 @@ The following `command line arguments`_ can also be used to modify the defaults:
 
 .. code-block:: shell
 
+     -r REVISION, --revision REVISION
+                           Git revision against which to compare the working tree
```

Comment:
I would add "should work with `HEAD~1`, `master..`, and other git expressions", or other similar phrasing, for people to understand it's not only hash based.

Reply:
Good idea. I expanded that as a suggestion in this review. Also did the corresponding thing in the argument parser.